eml_analyzer/cli_script.py

```diff
@@ -181,8 +180,27 @@ def extract_attachment(parsed_eml: Message, attachment_number: int, output_path:
     output_file = open(output_path, mode='wb')
     output_file.write(payload)
     info('Attachment extracted to {}'.format(output_path))
-
-
+    
+    
+def extract_all_attachments(parsed_eml: Message, output_path: str or None):
+    print_headline_banner('Attachment Extracting')
+    attachment = None
+    for child in parsed_eml.walk():
+      if child.get_filename() is None:
+        continue
+      print(child.get_filename())
```

Comment:
Maybe combine this print out with the print out in line 201: **info('Attachment extracted to {}'.format(path))**

Recommendation:
`info('Attachment "{}" extracted to {}'.format(attachment.get_filename(), path))`